projects/core/src/occ/adapters/product/converters/product-image-normalizer.spec.ts

```diff
@@ -37,6 +37,12 @@ describe('ProductImageNormalizer', () => {
         imageType: ImageType.PRIMARY,
         url: '/test2',
       },
+      {
+        altText: 'Test alt text',
+        format: 'zoom',
+        imageType: ImageType.PRIMARY,
+        url: 'https://hybris.com/demo.jpg',
```

Comment:
I'd at least also include a variation for `http` and  `//`, and the uppercase/lowercase variants. Even if you're code expects to start with http, a future refactor might change the code and the http variation might fail in production.